### DISCUSSION TITLE
Sudo in workflow - 13: Permission denied

### DISCUSSION THREAD

**Discussion by petrsx** [actions/actions-runner-controller#3186] [Questions]
Hi, i would like to install specific tool in the container however it is not possible without using sudo

This is working just fine on GH hosted runners

I have to say i tried almost every example i found here, however none solved the problem.
If someone can explain what is wrong or provide some example config, it will be great.

Thank you

**Comment by lukehsiao** [comment]
@petr-stupka, were you able to resolve this?

**Comment by petrsx** [comment]
Yes, i do not remember what i did, however it is no more an issue
